src/content-handlers/iiif/modules/uv-shared-module/css/scaffolding.less

```diff
@@ -6,8 +6,8 @@
 
 .uv-iiif-extension-host {
   *,
-  *:before,
-  *:after {
+  *::before,
```

Comment:
Here's more of this, if it matters -- though I'm beginning to suspect that maybe it doesn't. :-)